StudenSchool_DO/HW2/FileToRead.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Runtime.InteropServices;
+
+namespace HW2
+{
+    internal class FileToRead : IInformation
+    {
+        private const string Information = "Чтение";
+        private const string MessagesOperation = "Введите сколько строк хотите получить: ";
+        private const string PathWindows = @"..\..\..\Data\FileToRead.txt";
+        private const string PathOther = @"../../../Data/FileToRead.txt";
+
+        public string GetInformation()
+        {
+            return Information;
+        }
+
+        private string[] GetSpecifiedNumberOfLinesOfTextFromFile(int specifiedNumber)
+        {
+            string path = PathOther;
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+                path = PathWindows;
+
+            if (!File.Exists(path) || specifiedNumber == 0)
+                return new string[] {""};
+
+            string[] linesOfText = File.ReadAllLines(path);
+            return linesOfText[0..specifiedNumber];
```

Comment:
Hmm, you read all file and return only needed lines. Optional, if can, read only needed lines.